docs/README.md

```diff
@@ -0,0 +1,249 @@
+  cn "cc9ad6da-e05e-11e2-8e23-002590c3f078":
+    provision (image acc104a2-db81-4b96-b962-9b51409eadc0)
+Are you sure you want to proceed? (y/N): y
+
+service "garbage-collector": provisioning
+    server_uuid: 00000000-0000-0000-0000-002590c0933c
+     image_uuid: acc104a2-db81-4b96-b962-9b51409eadc0
+service "garbage-collector": provisioning
+    server_uuid: cc9ad6da-e05e-11e2-8e23-002590c3f078
+     image_uuid: acc104a2-db81-4b96-b962-9b51409eadc0
+service "garbage-collector": provisioned aefbf103-b815-4f95-84ea-e446c4e9bb50
+    server_uuid: 00000000-0000-0000-0000-002590c0933c
+     image_uuid: acc104a2-db81-4b96-b962-9b51409eadc0
+service "garbage-collector": provisioned a1f48ab2-6c8a-4538-8567-2ca74a9d5b4d
+    server_uuid: cc9ad6da-e05e-11e2-8e23-002590c3f078
+     image_uuid: acc104a2-db81-4b96-b962-9b51409eadc0
+[root@headnode (nightly-2) ~]#
+```
+
+At this point you should have `garbage-collector` instances but they will not
+yet be collecting any garbage. To have them start doing that, proceed to the
+next step.
+
+## Configuring garbage-collectors Shard Assignments
+
+There are 2 parts to updating the `garbage-collector` shard assignments. First
+you want to generate an assignment:
+
+```
+manta-adm gc gen-shard-assignment > gc-shard-assignment.json
+```
+
+you can take a look at this if you'd like to make sure it looks reasonable. It
+should look something like:
+
+```
+{
+    "aefbf103-b815-4f95-84ea-e446c4e9bb50": {
+        "GC_ASSIGNED_BUCKETS_SHARDS": [
+            {
+                "host": "1.buckets-mdapi.nightly.joyent.us",
+                "last": true
+            }
+        ],
+        "GC_ASSIGNED_SHARDS": [
+            {
+                "host": "1.moray.nightly.joyent.us",
+                "last": true
+            }
+        ]
+    },
+    "a1f48ab2-6c8a-4538-8567-2ca74a9d5b4d": {
+        "GC_ASSIGNED_BUCKETS_SHARDS": [
+            {
+                "host": "2.buckets-mdapi.nightly.joyent.us",
+                "last": true
+            }
+        ],
+        "GC_ASSIGNED_SHARDS": [
+            {
+                "host": "2.moray.nightly.joyent.us",
+                "last": true
+            }
+        ]
+    }
+}
+```
+
+which will mean that the first `garbage-collector` is responsible for the shards
+`1.buckets-mdapi` and `1.moray` and the second `garbage-collector` is
+responsible for the shards `2.buckets-mdapi` and `2.moray`. If this looks fine,
+you can apply the configuration by running:
+
+```
+manta-adm gc update gc-shard-assignment.json
+```
+
+which will apply the SAPI metadata changes in order to make the new
+configuration active. You should be able to confirm this by running:
+
+```
+manta-oneach -s garbage-collector "json dir_shards buckets_shards < /opt/smartdc/manta-garbage-collector/etc/config.json"
+```
+
+At this point garbage collection should be working for all deleted objects in
+both buckets and directory-style Manta.
+
+## Configuring garbage-collector Tunables
```

Comment:
Ah, I hadn't realized you were doc'ing these here. That other SAPI config vars doc (for MANTA-5091) can just point here.

Reply:
ok!